packages/cdk/apps/context/app.ts

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env node
 import { App } from "aws-cdk-lib";
 import "source-map-support/register";
-import { APP_NAME, APP_TAG_KEY, CONTEXT_TAG_KEY, PRODUCT_NAME, PROJECT_TAG_KEY, USER_EMAIL_TAG_KEY, USER_ID_TAG_KEY } from "../../lib/constants";
+import {
```

Comment:
Why import reformatted in this file, but not in "core/app.ts"? What is our convention on imports in TS?

Reply:
We use Prettier, which introduces line breaks once a line exceeds a specific length (160 chars?)